### PR TITLE
DOCS-7214 - Minor type fix discovered by translators

### PR DESCRIPTION
**Pull request by joel-hamill**
### Description 

https://confluentinc.atlassian.net/browse/DOCS-7214

Fix typo.

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

### CODE REVIEW

File: docs/overview.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ The use case is an |ak-tm| event streaming application that processes real-time
 
 The full event streaming platform based on |cp| is described as follows.
 Wikimedia's `EventStreams <https://wikitech.wikimedia.org/wiki/Event_Platform/EventStreams>`__ publishes a continuous stream of real-time edits happening to real wiki pages.
-A Kafka source connector `kafka-connect-sse <https://www.confluent.io/hub/cjmatta/kafka-connect-sse>`__ streams the server-sent events (SSE) from https://stream.wikimedia.org/v2/stream/recentchange, and a custom |kconnect| transform `kafka-connect-json-schema <https://www.confluent.io/hub/jcustenborder/kafka-connect-json-schema>`__ extracts the JSON from these messages and then are written to a |ak| cluster.
+A Kafka source connector `kafka-connect-sse <https://www.confluent.io/hub/cjmatta/kafka-connect-sse>`__ streams the server-sent events (SSE) from https://stream.wikimedia.org/v2/stream/recentchange, and a custom |kconnect| transform `kafka-connect-json-schema <https://www.confluent.io/hub/jcustenborder/kafka-connect-json-schema>`__ extracts the JSON from these messages and then the messages are written to a |ak| cluster.
```
Comment (ybyzek) [inline docs/overview.rst:22]: @joel-hamill For clarification, the extracted JSON is written; in contrast, the message w/out the JSON is NOT written.  So I think the PR is a little ambiguous on this.

Comment (joel-hamill) [reply docs/overview.rst:22]: @ybyzek okay can you suggest what this should be?

Comment (joel-hamill) [reply docs/overview.rst:22]: ```...and then the content is written to a |ak| cluster.```?

Comment (ybyzek) [reply docs/overview.rst:22]: 👍 SGTM

Comment (joel-hamill) [inline docs/overview.rst:22]:
```suggestion
A Kafka source connector `kafka-connect-sse <https://www.confluent.io/hub/cjmatta/kafka-connect-sse>`__ streams the server-sent events (SSE) from https://stream.wikimedia.org/v2/stream/recentchange, and a custom |kconnect| transform `kafka-connect-json-schema <https://www.confluent.io/hub/jcustenborder/kafka-connect-json-schema>`__ extracts the JSON from these messages and then the content is written to a |ak| cluster.
```

**Pull request merged**